src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -228,6 +223,8 @@ public Claim saveClaim(
                     new NotFoundException("Could not find claim with external ID '" + externalId + "'"));
         }
 
+        ccdEventProducer.createCCDClaimIssuedEvent(issuedClaim, authorisation);
```

Comment:
any reason why this is moved up ?

Reply:
It's because the event would otherwise get created before the document is generate.